### DISCUSSION TITLE
这个机器翻译是否受到社区的好评？

### DISCUSSION THREAD

**Discussion by lawrenz1** [kono-dada/Sakuranotoki-Chinese#46] [General]
我正在考虑为英文翻译做类似的项目  (サクラノ詩/刻），但英语社区对机器翻译有很大的反感。所以我很好奇中文社区是否也是如此，尽管GPT-4（和Claude Sonnet 3.5）的翻译质量比以前的方法要好得多。

如果这段话难以理解，我提前道歉。我是用Sonnet 3.5从英语翻译成中文的 :)

**Comment by kono-dada** [comment]
Of course this translation is well received by Chinese community, since it brought the game to the player much earlier. Occasionally it is even better than human’s translation. You see, it has gained 300+ stars.

**Reply by lawrenz1** [reply]
That is good to hear. We are in exciting times and I think most translations in the future will be done this way as MTL gets even better. (MTL first -> edit MTL until it is polished and ready for release)

**Reply by kono-dada** [reply]
that’s exactly how this translation processed the text
